ujt/transformers.py

```diff
@@ -373,3 +373,44 @@ def apply_slis_to_node_map(sli_list, node_map):
         node.slis.extend(new_node_slis)
 
     return node_map
+
+
+def sort_nodes_by_parent_relationship(elements):
```

Comment:
i forget, is there already a unit test for this method?

Reply:
Ah, I should probably write one for this method